cdap-app-fabric/src/test/java/co/cask/cdap/AppWithSchedule.java

```diff
@@ -50,8 +50,11 @@ public void configure() {
       scheduleProperties.put("anotherKey", "anotherValue");
       scheduleProperties.put("someKey", "someValue");
 
-      scheduleWorkflow(Schedules.createTimeSchedule("SampleSchedule", "Sample schedule", "0/1 * * * * ?"),
-                       "SampleWorkflow", scheduleProperties);
+      scheduleWorkflow(Schedules.builder("SampleSchedule")
```

Comment:
I'm not sure why they are separate. I suppose you could argue that they are separate because they are properties for the program and not for the schedule?  Perhaps in the future you could tie multiple programs to a single schedule, and then the properties for each program could be different?